tests/test_sorting.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import time 
+from utilities.driver import get_driver
+from pages.login_page import LoginPage
+from pages.product_page import ProductPage
+
+class TestSorting:
+    @pytest.fixture
+    def setup(self):
+        self.driver = get_driver()
+        self.driver.get("https://www.saucedemo.com/")
+        self.login_page = LoginPage(self.driver)
+        self.login_page.login("standard_user", "secret_sauce")
+        time.sleep(2)
+        yield
+        self.driver.quit()
+
+    def test_sort_name_a_to_z(self, setup):
+        product_page = ProductPage(self.driver)
+        product_page.sort_products("Name (A to Z)")
+        time.sleep(2)
```

Comment:
u can add the assertaion statement here or in the function as u want. but, u need to add it .